roles/lib_openshift/src/doc/sdnvalidator

```diff
@@ -0,0 +1,27 @@
+# flake8: noqa
+# pylint: skip-file
```

Comment:
These files contain variable assignments as well, and pragmas that are related to Python (`noqa`, `skip-file`).
Having the extension `.py` would enable automatic code highlighting on most text editors, make use of the pragmas to skip linting (without the extension I suspect pylint and flake8 don't even touch these files), and make it easier to figure out what the file is, and filter / find in the command line. E.g., we rely on the extension to find all Python files.